server/src/modules/users/infra/repos/auth-secret-repo/implementations/mikro-auth-secret-repo.ts

```diff
@@ -10,13 +10,13 @@ export class MikroAuthSecretRepo implements AuthSecretRepo {
   constructor(protected authSecretEntityRepo: EntityRepository<AuthSecretEntity>) {}
 
   async exists(clientId: string): Promise<Result<boolean, DBErrors>> {
-    const authSecretEntity = await this.authSecretEntityRepo.findOne({ 
-      clientId: clientId, 
+    const authSecretEntity = await this.authSecretEntityRepo.findOne({
```

Comment:
I remember there was a pretty hefty origin story behind `DBErrors` about how we'd handle errors from Mikro. Does that pattern require a try/catch here?

Reply:
Hmmm, Mikro doesn't seem to throw an error here. But in general, errors that could occur in db methods like this would have a DBError assigned.